tools/nixd.cpp

```diff
@@ -9,10 +9,6 @@
 
 int main() {
   std::error_code EC;
-  auto DebugOut = llvm::raw_fd_ostream("/tmp/nixd.out", EC,
```

Comment:
Currently we have initialized with the VSCode extension. So logging into `stderr` could be transferred into extension outputs. So delete hard-coded log file. (This should be another patch though)